### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Danny-imo**
### **In This Milestone:**

1. Use media queries to make sure that website displays correctly on desktop screens.
2. Lay out the elements on the page with Grid and Flexbox.
3. Make Use of Different background Images

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -482,4 +488,476 @@
...
+  }
+
+  .social {
+    font-family: "Poppins", sans-serif;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0;
+    text-align: left;
+  }
```
Comment (frempongdev) [inline style.css]: Good work here implementing most of the styling requirements from the Figma design. 🚀 However there is a tiny bit left to do to make it complete. ♻️ 

- [ ] Kindly make sure you stick to all the Figma design requirements as closely as possible including the `color`. Paying attention to design requirement will go a long way to help you satisfy clients in your future job.

Please check the designs below and make the due changes. 👍🏿 

### Your design ⬇️ 

image


### Figma requirements ⬇️ 

image

File: style.css [hunk 2]
```diff
@@ -482,4 +488,476 @@
...
+
+  .about-section {
+    display: flex;
+    gap: 75px;
+    border: 1px solid #cfd8dc;
+    padding: 20px;
+    border-radius: 16px;
+    width: 85%;
+    margin: 6%;
+  }
```
Comment (frempongdev) [inline style.css]: As earlier mentioned, please stick as closely as possible to the Figma design as possible. 🎯 

- [ ] I suggest you chack your padding and margin units and use responsive units like `%`, `em`, `rem` etc.

Kindly compare the images below and center your works as seen in the design template ♻️ 

### Your design ⬇️ 
image

### Figma design ⬇️ 
image

File: index.html [hunk 3]
```diff
@@ -75,69 +84,75 @@ <h4 class="about-head">Tonic</h4>
...
       <div class="canopy">
+        <h4 class="about-head">Uber Navigation</h4>
         <ul class="cano">
-          <li class="tec">CANOPY</li>
-          <li class="tech">Back End Dev</li>
-          <li class="tech">2015</li>
+          <li class="tec">UBER</li>
+          <li class="tech"> Lead Developer</li>
+          <li class="tech">2018</li>
         </ul>
```
Comment (frempongdev) [inline index.html:148]: - [ ] Please check and remove the extra space in your project details section. 👍🏿 

### Your design ⬇️ 
image


### Figma design ⬇️ 
image

File: index.html [hunk 4]
```diff
@@ -194,20 +209,20 @@ <h3>
...
 
   <div class="nrop">
-    <p>Frameworks <i class="fas fa-angle-right"></i> </p>
+    <p class="lang-hd">Frameworks <i class="fas fa-angle-right"></i> </p>
     <hr>
-    <p>Skills &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <i class="fas fa-angle-right"></i></p>
+    <p class="lang-hd">Skills &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <i class="fas fa-angle-right"></i></p>
     <hr>
   </div>
 </section>
```
Comment (frempongdev) [inline index.html:228]: Great work sticking to the design requirements so far. 🎨 I however have some few change suggestions to make the design complete. ♻️ 

Your webpage seem not to be completely responsive and has different margins in various screen sizes. In a desktop view, the `About` and `Contact` sections seem to be align to the left side of the page. 

about

- [ ] Please make sure that your sections are centered in all screen sizes as it ensures consistency in design, which is the core factor in responsive design. 🎯

File: style.css [hunk 5]
```diff
@@ -482,4 +488,476 @@
...
+    left: 16px;
+    top: 15px;
+    border: 1px solid #cfd8dc;
+  }
+
+  .name {
+    height: 48px;
+    width: 447px;
+    border: 1px solid #cfd8dc;
+  }
```
Comment (frempongdev) [inline style.css]: - [ ] Please check the design requirements provided by the Figma design and apply them to your form input placeholders. 🤝 
- [ ] Please reduce the space between the contact header text and the paragraph. ⏫ 

### Your design ⬇️ 
image

### Figma design ⬇️ 
image

**Review by frempongdev — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @Danny-imo   👋🏼 
I'm Frempong.

You've done a great job implementing most of the requirements for this project 🚀, but there are a few changes I recommend be made. You're almost done, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ No Linter errors. 🆗  
✅ Your project is documented professionally. 📜 


 ## Required Changes ♻️
_Kindly Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Alusp) [reply index.html:228]: - [ ] Great job in implementing the so far :clap: , However, As requested by the previous reviewer, Your webpage seem not to be completely responsive and has different margins in various screen sizes. In a desktop view, the About and Contact sections seem to be align to the left side of the page. making your website responsive for all screen sizes is essential for delivering an optimal user experience, maximizing reach, improving search engine visibility, saving time and resources, future-proofing your site, and fostering engagement and conversions. :slightly_smiling_face: 

Below is the screenshot of your design: :arrow_down: 

Student rough Design

File: index.html [hunk 6]
```diff
@@ -75,69 +84,75 @@ <h4 class="about-head">Tonic</h4>
...
-          sign-ups required
+          Experimental content creation feature that allows users to add to an existing story over 
+          the course of a day without spamming their friends.
         </p>
         <ul class="tech1">
           <li class="tech-t">HTML</li>
+          <li class="tech-r">Ruby On Rails</li>
           <li class="tech-t">CSS</li>
           <li class="tech-j">JavaScript</li>
         </ul>
```
Comment (Alusp) [inline index.html:113]: [OPTIONAL] Please ensure that you give some breathing space in all your  `tech1` class items, below is an Illustration between yours and the given design. :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

S design 2

</td>
      <td>

Figma Design 2

</td>
    </tr>
  </table>

File: index.html (hunk 3 above)
Comment (Alusp) [reply index.html:148]: [OPTIONAL] Please ensure that you always match your design according to the given design kindly check your `Uber` font weight, Consistency is crucial in design, especially when working on a larger project or within an established brand. Matching your design to the given design ensures a cohesive visual identity across different components, screens, or materials. Consistency helps create a unified experience for users and maintains brand recognition and professionalism. :slightly_smiling_face: 

Below is an example of the difference: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

s uber navigation

</td>
      <td>

Figma Design navigation

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @Danny-imo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 
close

## Highlights

- [ ] Good Commit History :green_circle: 
- [ ] Good Github Flow :green_circle: 
- [ ] Good PR Summary :green_circle: 
- [ ] No Linter Errors :green_circle: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**